styles.css

```diff
@@ -80,18 +78,18 @@ a:active {
 }
 
 #contact {
-  border-top: 2px solid #878787;
   border-bottom: 2px solid #878787;
+  border-top: 2px solid #878787;
 }
 
 /* Floated Columns -----------------*/
 
 /* Float Clearfix ------------------ */
 
 .group:after {
-  content: "";
-  display: table;
   clear: both;
+  content: '';
+  display: table;
 }
```

Comment:
no changes, just some sorting.
This is the vs code extension I use for sorting sections of code.
<img width="554" alt="Screen Shot 2020-12-19 at 3 09 06 PM" src="https://user-images.githubusercontent.com/23409625/102698549-30c21a80-420c-11eb-958b-ab70c3796a94.png">

Reply:
Cool

Reply:
I don't know what this means - your comment from above. 

"I recommend closing old PRs if you don't merge them like this one #1
And consider deleting the branch after closing the PR."